cytominer_database/utils.py

```diff
@@ -25,7 +25,13 @@ def find_directories(directory):
     :return: list of subdirectories of ``directory``
 
     """
-    return glob.glob(os.path.join(directory, '*/'))
+    directory = os.path.abspath(directory)
```

Comment:
Why was this needed? @gwaygenomics

Reply:
all changes to path arguments here were me troubleshooting the windows build 😅

Reply:
🥇 for trying!